src/main/java/data_gateway/CalendarManager.java

```diff
@@ -13,6 +16,12 @@
 
     List<EventReader> getAllEvents();
 
+    void updateName(long id, String newName);
+
+    void updateStartTime(long id, LocalTime newStartTime);
+
+    void updateEndTime(long id, LocalTime newEndTime);
+
```

Comment:
Can we also have an `updateTags()` method? I think you would have to change `tags` in the `Event` class to become not final too